pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -521,6 +532,49 @@ func (l LibvirtWrapper) SetupLibvirt() (err error) {
 	return nil
 }
 
+func getLibvirtLogsFilters() (logFilters string, enableDebugLogs bool) {
+
+	_, envVarExists := os.LookupEnv(services.ENV_VAR_LIBVIRT_DEBUG_LOGS)
+	if !envVarExists {
+		return "", false
+	}
+	launcherVerbosityLevelEnvVar, envVarExists := os.LookupEnv(services.ENV_VAR_VIRT_LAUNCHER_LOG_VERBOSITY)
+	if !envVarExists {
+		return "", false
+	}
+
+	libvirtLogVerbosity, err := strconv.Atoi(launcherVerbosityLevelEnvVar)
+	if err != nil {
+		log.Log.Infof("cannot apply LIBVIRT_DEBUG_LOGS value %s - must be a number", launcherVerbosityLevelEnvVar)
+		return "", false
+	}
+
+	if libvirtLogVerbosity < 5 {
+		libvirtLogVerbosity = 5
+	}
+
+	const LogsLevel8 = "3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access"
+	const LogsLevel7 = LogsLevel8 + " 3:util.threadjob 3:cpu.cpu"
+	const LogsLevel6 = LogsLevel7 + " 3:qemu.qemu_monitor"
+	const LogsLevel5 = LogsLevel6 + " 3:qemu.qemu_monitor_json 3:conf.domain_addr"
+	const allowAllOtherCategories = " 1:*"
+
+	switch libvirtLogVerbosity {
+	case 5:
+		logFilters = LogsLevel5
+	case 6:
+		logFilters = LogsLevel6
+	case 7:
+		logFilters = LogsLevel7
+	case 8:
+		logFilters = LogsLevel8
+	default:
+		logFilters = LogsLevel8
```

Comment:
If the behavior for 8 and the default are going to be the same, then the `case 8` bit can be left out I think.

Reply:
I think that:
* If one day we want to change the default behavior we wouldn't need to create another case. Not creating another case for 8 explicitly might cause problems. Therefore this approach is less error-prone
* Easier to read and understand (in my personal opinion)
* Does not have any effect on performance